### DISCUSSION TITLE
How to get actives page's complete page tree as breadcrumbs.

### DISCUSSION THREAD

**Discussion by sujansiddarth** [squidfunk/mkdocs-material#4099] [Q&A]
Hello, Here in the breadcrumbs as of now, I could only see the active page's name..But How to get the complete page's tree of that actives page as breadcrumbs...please need help on this..How do I get the breadcrumbs as introduction>intro1 instead of only intro1...Attached screenshots for reference. @squidfunk 
![Screenshot from 2022-07-06 17-56-16 (1)](https://user-images.githubusercontent.com/64825547/177782158-01e44ee5-5268-45d1-9bc0-b73d6327453e.png)
![image (1)](https://user-images.githubusercontent.com/64825547/177782200-e31e709c-1583-47e6-86cf-c48cf5a54cb4.png)

**Comment by squidfunk** [comment]
Thanks for asking. I'm not sure where the thing you marked on the picture is coming from. Did you override the theme and change the templates? If so, [this](https://github.com/mkdocs/mkdocs/issues/693) might help.

**Reply by milovanovicdusan** [reply]
@squidfunk — I am experiencing a problem with _breadcrumbs_ when the `nav` section is specified in the `mkdocs.yml` file. In that case, _breadcrumbs_ disappear. When no `nav` section exists, the _breadcrumbs_ configuration (`navigation.path`) works fine. Is it possible to have breadcrumbs work when the navigation is configured?

**Reply by squidfunk** [reply]
Thanks for asking. There should be no such limitations. Could I please ask you to file a proper issue with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) that clearly showcases the problem? Please also carefully read our [bug reporting guide](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) – it's likely a quick fix for us then ☺️

**Reply by milovanovicdusan** [reply]
Thanks for the quick reply @squidfunk ! I've just submitted new issue, number https://github.com/squidfunk/mkdocs-material/issues/8199.

I tried to provide only the minimal configuration to reproduce the bug. Still, the automatic creation of the `.zip` file using the built-in `info` `plugin` resulted in a way-larger-than maximum allowed size for attaching the file to the issue report - 42.4 MB vs. maximum 1 MB. Please let me know if I can send the file by email or otherwise. I provided detailed, stepwise reproduction description of the issue that might help in the absence of the file.

**Reply by squidfunk** [reply]
Please do not double post the exact same message. We monitor all inquiries, so this is not necessary. It just creates noise and makes more work for all of us, including everybody that posted to an issue or discussion before.
